ndarray/ndarray-tfjs/src/jsMain/kotlin/io.kinference.ndarray/core/Memory.kt

```diff
@@ -8,3 +8,20 @@ import io.kinference.ndarray.arrays.ArrayTFJS
 external fun tidy(nameOrFn: () -> Array<ArrayTFJS>, fn: (() -> Array<ArrayTFJS>)?): Array<ArrayTFJS>
 
 external fun tidy(nameOrFn: String, fn: (() -> Array<ArrayTFJS>)?): Array<ArrayTFJS>
+
+external class Engine {
```

Comment:
Maybe rename it? We already have engines as backends in KInference and this naming adds some ambiguity